packages/desktop-client/src/components/accounts/Balance.tsx

```diff
@@ -16,10 +17,23 @@ import { Text } from '../common/Text';
 import { View } from '../common/View';
 import { PrivacyFilter } from '../PrivacyFilter';
 import { CellValue, CellValueText } from '../spreadsheet/CellValue';
+import { type SheetFields, type SheetNames } from '../spreadsheet/index';
 import { useFormat } from '../spreadsheet/useFormat';
 import { useSheetValue } from '../spreadsheet/useSheetValue';
 
-function DetailedBalance({ name, balance, isExactBalance = true }) {
+import { type ReconcilingMessage } from './Reconcile';
+
+type DetailedBalanceProps = {
+  name: string;
+  balance: number | null;
+  isExactBalance: boolean;
+};
+
+function DetailedBalance({
+  name,
+  balance,
+  isExactBalance = true,
```

Comment:
`isExactBalance` is intentionally and explicitly set to `true` because that was is what the original author intended.